manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -109,8 +135,68 @@ public void saveImageFile(String path, FileInfo fileInfo) {
         try (OutputStream out = new FileOutputStream(file)) {
             out.write(CodecUtil.decodeBase64(fileInfo.getContents()));
         } catch (IOException | SecurityException exception) {
-            LOG.log(Level.WARNING, "Error when saving image in manger_config.json", exception);
+            LOG.log(Level.WARNING, "Error when saving image in manager_config.json", exception);
```

Comment:
I know it wasn't there before but this should throw an exception back to the reponse (500 server error)